assets/js/admin/woocommerce_admin.js

```diff
@@ -32,10 +32,10 @@ jQuery( function ( $ ) {
 			$( '.wc_error_tip' ).fadeOut( '100', function() { $( this ).remove(); } );
 		})
 
-		.on( 'change', '.wc_input_price[type=text], .wc_input_decimal[type=text], .wc-order-totals #refund_amount[type=text]', function() {
+		.on( 'change', '.wc_input_price[type=text], .wc_input_decimal[type=text]', function() {
 			var regex;
 
-			if ( $( this ).is( '.wc_input_price' ) || $( this ).is( '#refund_amount' ) ) {
+			if ( $( this ).is( '.wc_input_price' ) ) {
```

Comment:
Since this is being refactored, I'd be using `hasClass()` instead of `is()`. `is()` is slower and isn't necessary for checks of this kind.